examples/demo/src/i18n/en.js

```diff
@@ -51,10 +51,16 @@ export default {
                 orders: 'Orders',
                 reviews: 'Reviews',
                 stats: 'Stats',
+                password: 'Password',
+                change_password: 'Change Password',
             },
             page: {
                 delete: 'Delete Customer',
             },
+            errors: {
+                password_mismatch:
```

Comment:
This message is missing in the french translations

Reply:
Done now :+1: